syntax = "proto3";
package ProjectJ;

import "Enum.proto";
import "Struct.proto";

message C_VERIFY_TOKEN
{
	int32 account_id = 1;
	string token = 2;
}

message S_VERIFY_TOKEN
{
	bool result = 1;
}